src/foundry/common/data/data.mjs/cardData.d.ts

```diff
@@ -0,0 +1,249 @@
+import type {
+  ConfiguredData,
+  ConfiguredFlags,
+  ConfiguredSource,
+  FieldReturnType,
+  PropertiesToSource
+} from '../../../../types/helperTypes.js';
+import type DocumentData from '../../abstract/data.mjs.js';
+import * as fields from '../fields.mjs';
+import type { CardFaceData, CardFaceDataConstructorData } from './cardFaceData.js';
+
+interface CardDataSchema extends DocumentSchema {
+  _id: typeof fields.DOCUMENT_ID;
+  name: typeof fields.REQUIRED_STRING;
+  description: typeof fields.BLANK_STRING;
+  type: FieldReturnType<
+    typeof fields.REQUIRED_STRING,
+    {
+      default: () => string;
+      validate: (t: unknown) => boolean;
+      validationError: '{name} {field} "{value}" is not a valid type';
+    }
+  >;
+  data: fields.SystemDataField;
+  suit: typeof fields.BLANK_STRING;
+  value: typeof fields.NUMERIC_FIELD;
+  back: {
+    type: typeof CardFaceData;
+    required: true;
+    default: Record<string, never>;
```

Comment:
Why not `{}`?

Reply:
Because `{}` is a different type than `Record<string, never>`. As eslint likes to explain, `{}` means any non-nullish value.